doc/versioning.rst

```diff
@@ -82,3 +82,14 @@ Example
   | oneAPI provisional 1.1 rev 2
   | oneAPI provisional 1.1 rev 3
   | oneAPI 1.1 rev 1
+
+Programmatic versioning
+=======================
+
+The specification version as mentioned above can be used programatically by appending all digits of the specification version in the format of <MAJOR><MINOR><REVISION>. MINOR and REVISION version use two digits. This versioning can be used to check the compatibility of the implementation with the specification version.
```

Comment:
"appending all digits" to what? You are not proposing a macro/variable so I do not understand where these digits go.

Reply:
Added macro names.